db/Row.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable typescript-sort-keys/interface */
+export type CsvRow = {
+  account_id: string;
+  date: string;
+  block_timestamp: number;
+  block_height: number;
+  transaction_hash: string;
+  // transaction_type: string;
+  from_account: string;
+  to_account: string;
+  amount_transferred: number;
+  ft_amount_in: string;
+  ft_currency_in: string;
+  ft_amount_out: string;
+  ft_currency_out: string;
+  method_name: string;
+  amount_staked: number;
+  args: string;
+};
+
+export type ArgsJson = {
+  amount: number;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  actions: any[]; // TODO: Improve the type.
+  receiver_id?: string;
+  msg?: string;
+};
+
+export type Args = {
+  method_name: string;
+  deposit: number;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  args_base64: any; // TODO: Improve the type.
+  args_json: ArgsJson;
+};
+
+export type IndexerRow = {
```

Comment:
Are these all the rows incoming from indexer? I had prior understanding there were many more

Reply:
Did you mean to ask "Are these all the _fields_ incoming from indexer?"

I think these were the only ones that I've seen us use in this repo so far:

```
  account_id: string;
  receiver_account_id: string;
  receipt_predecessor_account_id: string;
  receipt_receiver_account_id: string;
  block_timestamp: number;
  block_height: number;
  transaction_hash: string;
  action_kind: string;
  args?: Args;
  ```

But I'm also noticing problems and created this ticket that I'm working on: https://near-foundation.atlassian.net/browse/TTA-63

Reply:
Yes, I meant values/fields ! There are more from my understanding. See https://github.com/khorolets/near-indexer-for-explorer/blob/master-lake/docs/near-indexer-for-explorer-db.png for context on explorer mainnet table columns which are getting queried. 

What is the major benefit of defining IndexerRow?

Reply:
Yep, that's a helpful schema image that I've been referencing.

And see https://github.com/NEARFoundation/transaction-tracking-app/pull/19/files#diff-50e534ccae2acc10c6940485516584865d5a9bb5ef22dbd539e4cd92e0aeafc7R2 

We'll want to whittle that down and not pull in as many fields as we currently are.

The benefit of creating any types is clarity. E.g. we have multiple functions accepting an indexer row as an argument. It's helpful to define what that means. What does it need to contain.

Even the 3 of us are still figuring that out because it's not obvious yet.  So we'll definitely want it to be clear (for ourselves in the future and for anyone else who works on this).